Lib/fontbakery/checks/base_has_width.py

```diff
@@ -36,7 +42,7 @@ def check_base_has_width(font, config):
         if codepoint == 0 or codepoint is None:
             continue
 
-        if advance == 0 and not gid not in mark_glyphs(font.ttFont):
```

Comment:
yeah! This double usage of the `not` operator looks like a silly coding mistake, indeed!